docs/low-code-sql/data-tests/data-tests.md

```diff
@@ -9,15 +9,22 @@ tags:
   - sql
 ---
 
-You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project by querying the single table result generated from the transformation steps shown.
+You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project.
+
+For example, find below a **Test** named `ref_int_orders_customers` which checks the validity of the SQL in the `HelloWorld_SQL` Project. In particular, the referential integrity check for orders and customers asserts that every `customer_id` entry in the `orders` table is present in the `customers` table.
 
 ![Project test canvas](img/project-test-canvas.png)
+This test starts with several **(1) models** from the `HelloWorld_SQL` Project, combines their data with a series of **(2) transformation** steps, and the resulting table is fed into the **(3) Data Test** Gem.
```

Comment:
If you decide to show the image here, I suggest explaining in this doc. 

Alternatively, this doc can be more of an overview, where we don't show (or explain) the referential integrity test since that's explained in the "Use Project Test" sub-page.

Reply:
This is a good point. Added the explanation.